ansible/pulp-from-source.yml

```diff
@@ -16,6 +16,8 @@
           This playbook requires Ansible 2.4+. Consider creating a virtualenv
           and installing Ansible with pip.
   roles:
+    - role: configure_rhel_7
+      when: ansible_distribution == "RedHat" or ansible_distribution == "CentOS"
```

Comment:
This is broken, because it will try to use the Python 3 interpreter.

Reply:
There's no way to avoid that apart from taking the manual step to change ```ansible_python_interpreter``` in the ```group_vars/all``` file, which I left a note up above to do.

Pulp-CI avoids this by manually specifying that variable on the command line, so nothing changes on that front.

Reply:
The ideal solution here is to set this variable in `host_vars` or `group_vars`, but since we don't version-control an inventory file, we can't use the ideal solution. Setting this varible on the command line seems like the next best option.

I agree that running two playbooks is not a great solution. It works, but it'd be far better to have just one. Given that this one playbook calls both the `configure_rhel_7` role and all the other roles, it'd probably be good to drop the `configure-rhel-7.yml` playbook in the future. The Jenkins configurations need to be updated first, though.

Reply:
@Ichimonji10 Are those concerns blockers to this PR?

I'm not sure that it would be worth the effort to try to dynamically populate ansible "facts" or an inventory file, etc. based on what image is being used.  I don't suspect we'll actually need a CentOS development environment too frequently (unless it's something QE is interested in).  As far as CI goes, overriding ```ansible_python_interpreter``` on the command line is something that can be done once and and then left alone.

Reply:
FWIW, I did attempt a runtime solution where, before the python parts of the ansible scripts would run, it would attempt to install the normal python3 package on CentOS so that Ansible could use the same interpreter.

Unfortunately the latest versions of Ansible don't support Python 3.4 so that didn't end up working.

Reply:
> FWIW, I did attempt a runtime solution where, before the python parts of the ansible scripts would run, it would attempt to install the normal python3 package on CentOS so that Ansible could use the same interpreter.
>
> Unfortunately the latest versions of Ansible don't support Python 3.4 so that didn't end up working.

Agreed. Plus, using the Python 2.7 interpreter that ships with RHEL 7 and CentOS 7 is the most low-impact solution. I dislike using a more complicated solution (install a second interpreter) when a simpler, lower-impact solution (use the existing interpreter) is available.

Reply:
> Are those concerns blockers to this PR?

I didn't list any concerns.